tests/wicked/startandstop/ref/t08_sit_tunnel_ifdown.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: SIT tunnel - ifdown
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use base 'wickedbase';
+use strict;
+use testapi;
+use lockapi;
+
+sub run {
+    my ($self) = @_;
+    record_info('Info', 'SIT tunnel - ifdown');
+    $self->create_tunnel_with_commands('sit1', 'sit', '127');
+    mutex_wait('test_sit_tunnel_ifdown_ready');
+}
+
+sub test_flags {
+    return {always_rollback => 1};
+}
+
+sub post_run_hook {
+    mutex_wait("test_sit_tunnel_ifdown_ready");
+}
+
+sub post_fail_hook {
+    mutex_wait("test_sit_tunnel_ifdown_ready");
```

Comment:
Doesn't we wait for the mutex 'test_sit_tunnel_ifdown_ready' twice? I would remove the post_run_hook() and post_fail_hook() here.

Reply:
Yes, it's twice because I forgot to remove it form run(). I would like to keep it in post_run_hook and post_fail_hook as this script contains things that can fail. Same as all the tests in "sut" directory.